pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -152,6 +152,18 @@
     dependencies = with super; [ plenary-nvim ];
   };
 
+  avante-nvim = (callPackage ./avante-nvim { }).overrideAttrs {
+    dependencies = with self; [
+      copilot-lua
+      dressing-nvim
+      img-clip-nvim
+      nvim-web-devicons
+      nui-nvim
+      plenary-nvim
+      render-markdown
```

Comment:
i think both render-markdown and img-clip-nvim are optional dependencies, no?

Reply:
According to
https://github.com/yetone/avante.nvim/blob/7532e590d28c6fffe88aa17ba6689c4ec8fec8ba/README.md
these should be optional:
```
copilot-lua
img-clip-nvim
nvim-web-devicons
render-markdown
```

But i get an error on nvim startup if img-clip-nvim is not installed.

I guess we can remove the other three.
We don't have a mechanism for specifying optional dependencies, right?